lessons/modelevaluation.ipynb

```diff
@@ -0,0 +1,786 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "In our linear model fit example, we can compute the evidence for the linear and quadratic models, and form the odds ratio $R$.\n",
+    "\n",
+    "```\n",
+    "log Evidence for Straight Line Model: -157.2\n",
+    "log Evidence for Quadratic Model: -120.7\n",
+    "Evidence ratio in favour of the Quadratic Model:\n",
+    "  7e15 to 1\n",
+    "```\n",
+    "\n",
+    "The 26 unit difference in log evidence between the two models translates to a _huge_ odds ratio in favour of the quadratic model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "Incidentally those data did not come from *either* a linear or quadratic model..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "The same Jeffreys scale used to interpret the information criteria can be used to interpret evidence ratios:\n",
+    "\n",
+    "| $R$    | Strength of Evidence for Model 2 |\n",
+    "|--------|:------------------------:|\n",
+    "|  < 1   | Negative |\n",
+    "|  1-3   | Barely worth mentioning |\n",
+    "|  3-10  | Substantial |\n",
+    "| 10-30  | Strong |\n",
+    "| 30-100 | Very strong |\n",
+    "|   >100 | Decisive |\n",
+    "\n",
+    "> The Bayesian Information Criterion (BIC) is an approximation of $R$ (assuming $N$ datapoints greatly outnumber $k$ parameters, and the priors are uninformative)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "**Calculating the evidence**\n",
+    "\n",
+    "Estimates directly calculated from Markov chains produced for parameter inference are generally not reliable.\n",
+    "\n",
+    "Good methods include nested sampling (e.g. [MultiNest](https://github.com/JohannesBuchner/PyMultiNest)) and parallel tempering (e.g. [emcee](http://dan.iel.fm/emcee/current/))."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "**Bayesian Evidence: closing thoughts**\n",
+    "\n",
+    "* The Bayesian evidence is *qualitatively different* from other model assessments. While they focus primarily on *prediction accuracy,* the evidence is the way in which information from the prior PDF propagates through into our posterior beliefs about the model as a whole.\n",
+    "\n",
+    "* There are no inherent mathematical limitations to its use, in contrast to various other hypothesis tests that are only valid under certain assumptions (such as the models being nested, e.g. the classical $F$ test for comparing $\\chi^2$ values). _Any two models can be compared and the odds ratio computed._"
+   ]
+  }
+ ],
+ "metadata": {
```

Comment:
Suggest a summarizing slide covering the 3 broad techniques in this chunk, and emphasizing that they address subtly different questions.

Reply:
Good idea. Will return to the three questions at the end.Thanks!